test/protocol/clients/BosonVoucherTest.js

```diff
@@ -1467,6 +1474,195 @@ describe("IBosonVoucher", function () {
+        // Approve another address to transfer the voucher
+        await bosonVoucher.connect(operator).setApprovalForAll(rando2.address, true);
+
+        await expect(
+          bosonVoucher.connect(rando2).transferPremintedFrom(operator.address, rando.address, offerId, tokenId, "0x")
+        )
+          .to.emit(bosonVoucher, "Transfer")
+          .withArgs(operator.address, rando.address, tokenId);
+      });
+
+      context("💔 Revert Reasons", async function () {
+        it("Transfer preminted voucher, but from is not the contract owner", async function () {
+          await expect(
+            bosonVoucher.connect(rando).transferPremintedFrom(rando.address, buyer.address, offerId, tokenId, "0x")
+          ).to.be.revertedWith(RevertReasons.NO_SILENT_MINT_ALLOWED);
+        });
+
+        it("Cannot transfer preminted voucher twice", async function () {
+          // Make first transfer
+          await bosonVoucher
+            .connect(operator)
+            .transferPremintedFrom(operator.address, buyer.address, offerId, tokenId, "0x");
+
+          // Second transfer should fail, since voucher has an owner
+          await expect(
+            bosonVoucher
+              .connect(operator)
+              .transferPremintedFrom(operator.address, rando.address, offerId, tokenId, "0x")
+          ).to.be.revertedWith(RevertReasons.NOT_COMMITTABLE);
+
+          // It should also fail if transfer done with standard safeTransferFrom
+          await expect(
+            bosonVoucher
+              .connect(operator)
+              ["safeTransferFrom(address,address,uint256,bytes)"](operator.address, rando.address, tokenId, "0x")
+          ).to.be.revertedWith(RevertReasons.ERC721_CALLER_NOT_OWNER_OR_APPROVED);
+        });
+
+        it("Transfer preminted voucher, which was committed and burned already", async function () {
+          await bosonVoucher
+            .connect(operator)
+            .transferPremintedFrom(operator.address, buyer.address, offerId, tokenId, "0x");
+
+          // Redeem voucher, effectively burning it
+          await setNextBlockTimestamp(ethers.BigNumber.from(voucherRedeemableFrom).toHexString());
+          await exchangeHandler.connect(buyer).redeemVoucher(tokenId);
+
+          // Transfer should fail, since voucher has been burned
+          await expect(
+            bosonVoucher
+              .connect(operator)
+              .transferPremintedFrom(operator.address, rando.address, offerId, tokenId, "0x")
+          ).to.be.revertedWith(RevertReasons.NOT_COMMITTABLE);
+        });
+
+        it("Transfer preminted voucher, which was not committed but burned already", async function () {
+          // Void offer
+          await offerHandler.connect(operator).voidOffer(offerId);
+
+          // Burn preminted vouchers
+          await bosonVoucher.connect(operator).burnPremintedVouchers(offerId);
+
+          // None of reserverd but not preminted tokens should have an owner
```

Comment:
```suggestion
          // None of reserved but not preminted tokens should have an owner
```

Reply:
Fixed here and in other places with same mistake